.github/workflows/ci.yml

```diff
@@ -8,6 +8,9 @@ permissions:
   contents: read
   actions: write
 
+env:
+  NODE_VERSION: '14'
```

Comment:
Just extracting it to an env var so we only need to change it in a single place in the future.

Reply:
That's a lot better if we need to change it later! Good catch!